### PR TITLE
[12.0][MIG] document_page_project

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: document_page_project/models/__init__.py [hunk 1]
```diff
@@ -1,2 +1,3 @@
-from . import project_project
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```
Comment (LoisRForgeFlow) [inline document_page_project/models/__init__.py:1]: Why do you readd license in init files? I think it is useless

File: document_page_project/__manifest__.py [hunk 2]
```diff
@@ -3,12 +3,12 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 {
-    'name': 'Project Wiki',
+    'name': 'Document Page Project',
```
Comment (LoisRForgeFlow) [inline document_page_project/__manifest__.py:6]: I liked my original name :cry:

**Review by LoisRForgeFlow — APPROVED** [review]
Minor comments

File: document_page_project/__manifest__.py (hunk 2 above)
Comment (MiquelRForgeFlow) [reply document_page_project/__manifest__.py:6]: 😂

File: document_page_project/models/__init__.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply document_page_project/models/__init__.py:1]: I saw a lot of people putting the license :S

File: document_page_project/__manifest__.py (hunk 2 above)
Comment (MiquelRForgeFlow) [reply document_page_project/__manifest__.py:6]: Still exists: https://github.com/OCA/knowledge/pull/209/files#diff-3fb747d9f98468a02de9776e52a03905R16

**Comment by rvalyi** [comment]
Hello, I don't get it, hasn't this been already done in this merged PR https://github.com/OCA/knowledge/pull/208 ??

**Comment by rvalyi** [comment]
my bad, not the same module, deleting my comments sorry!

File: document_page_project/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+{
+    'name': 'Document Page Project',
+    'summary': 'This module links document pages to projects',
+    'version': '12.0.1.0.0',
+    "development_status": "Beta",
+    'category': 'Project',
+    'author': 'Eficent, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/project',
```
Comment (tbaden) [inline document_page_project/__manifest__.py]: isn't it https://github.com/OCA/knowledge ?

Comment (JordiBForgeFlow) [reply document_page_project/__manifest__.py]: @tbaden  fixed!

**Review by tbaden — APPROVED** [review]

**Pull request merged**